creativecloud/scripts/utils.js

```diff
@@ -86,9 +98,21 @@ function getDecorateAreaFn() {
         import(`${getConfig().codeRoot}/deps/interactive-marquee-changebg/changeBgMarquee.js`);
         break;
       }
-      case firstBlock?.classList.contains('marquee'):
-        firstBlock.querySelectorAll('img').forEach(eagerLoad);
+      case firstBlock?.classList.contains('marquee'): {
+        // Load image eagerly for specific breakpoint
+        const viewport = defineDeviceByScreenSize();
+        const lcpImageByViewport = {
+          MOBILE: 'div:first-child img',
+          TABLET: 'div:nth-child(2) img',
+          DESKTOP: 'div:nth-child(3) img',
+        };
+        const backgroundImages = firstBlock.querySelector('div:nth-child(1)');
+        if (backgroundImages.querySelectorAll('img').length === 1) eagerLoad(firstBlock.querySelector('div:first-child img'));
```

Comment:
Nit: For all of the `:nth-child(1)` and `:first-child` selectors: `querySelector` takes the first match automatically so are these necessary?

Reply:
That makes sense. I had it initially as I was using the selectors used in milo-college. I have updated the code.